Ginger/Ginger/Actions/ActionConversion/ActionsConversionWizard.cs

```diff
@@ -154,8 +154,8 @@ public async void ProcessConversion(ObservableList<BusinessFlowToConvert> lst, b
                     foreach (var bf in lst)
                     {
                         if (bf.IsSelected)
-                        {
-                            bf.BusinessFlow.RestoreFromBackup(true);
+                        {                            
+                            bf.BusinessFlow.RestoreFromBackup(clearBackup: false);
```

Comment:
please explain why we need to keep the backup?

Reply:
we have kept the "Re-Convert" button visible and user can have the option to reconvert it multiple times, if suppose user have selected all the businessflows and reconvert them successfully and if user again clicks to reconvert button then there will not be any backup and nothing will be converted, if this is ok then we can remove the backup